lib/pipeline.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
```

Comment:
This is because of line 60 and 64, to update the job. We don't do this for other files.